code/game/machinery/computer/records/security.dm

```diff
@@ -216,6 +218,8 @@
 		investigate_log("New Crime: <strong>[input_name]</strong> | Added to [target.name] by [key_name(user)]. Their previous status was [target.wanted_status]", INVESTIGATE_RECORDS)
 		target.wanted_status = WANTED_ARREST
 
+		update_all_security_huds()
```

Comment:
Already answered this in my reply to Melbert. Reposting: 

> And yea that is where I got it from, was already in the code. If I am correct (big IF), I think the issue is that when you are setting someone with the console, there isn't a direct mob attached to that crew record that you can just visually update, rather anyone (or even multiple people at once) could be attached to that record depending on what ID they are wearing, what their face looks like, etc. So the simplest way to handle it is to just update the sechuds on everybody. Given that using a records console is a pretty rare action I don't think there should be even a slightly noticeable impact on performance.

Actually now that I think about it, when you are directly setting someone with sechuds examine menu, currently it just updates the sechud visuals on that one mob. But there could still be other mobs who are connected to that record so it would probably make sense to update everyone. The time this is most noticeable is with paradox clones, if you use sechud examine menu to set one, the record on both will update, but the visuals will only update on the one you clicked, unless the other one does another trigger that updates their sechud visuals. 

If you have thoughts on a better way to handle this, I'm all ears. Otherwise I'm gonna go ahead and change the sechud examine menu code to fix this as well.

Reply:
So I went ahead and did that optimization, as well as fixing the sechud examine menu thing and fixing another bug I found while testing (when you void someone's last crime and it automatically marks them as discharged, that wasn't updating huds) and I also added documentation to the 2 new procs I made.